src/components/pages/lessons/overlay/add-note-overlay.tsx

```diff
@@ -0,0 +1,93 @@
+import * as React from 'react'
+import {Switch} from '@headlessui/react'
+import VisuallyHidden from '@reach/visually-hidden'
+
+const AddNoteOverlay: React.FC<{sendPlayerState: any}> = ({
+  sendPlayerState,
+}) => {
+  const [enabled, setEnabled] = React.useState(false)
+  const inputRef = React.useRef<HTMLTextAreaElement>(null)
+  React.useEffect(() => {
+    if (inputRef) {
+      inputRef.current?.focus({preventScroll: true})
+    }
+  }, [])
+  return (
+    <div
+      className="w-[34rem] h-[20rem] rounded-md bg-white p-4 flex flex-col"
+      id="add-note-overlay"
+    >
+      <div className="flex-shrink-0 flex justify-end">
+        <button
+          type="button"
+          onClick={() => sendPlayerState('VIEW')}
```

Comment:
this might be `onClose` versus `onAddNote` described below, but idea is the same. Send in a click handler function, handle click logic in player level to change state there not passing xstate around.